css/index.css

```diff
@@ -1,3 +1,560 @@
 /* Use your own code or past solution for Great Idea Web Page CSS here! */
 
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
 
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+/*
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+	margin-bottom: 15px;
+	
+}
+*/
+
+/* Your code starts here! */
+.container {
+	width: 900px; 
+    margin: auto;
+	
+}
+  
+
+header nav {
+ display: flex;
+  margin-top: 35px;
+  
+}
+
+.logo {
+	margin-left:650px;
+	display: flex;
+	margin-top: -50px;
+
+	
+}
+
+header nav a {
+	margin: 20px; 
+	
+}
+
+
+
+.sub img{
+display: flex;
+margin-left: 450px;
+    margin-bottom: 50px;
+	margin-top: -290px;
+}
+
+
+
+.sub h1 {
+	font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+	display:flex;
+	margin-top: 30px;
+	font-size: 60px;
+	text-align: center;
+	margin-left: 100px;
+	margin-top:100px;
+}
+
+.sub h3 {
+	border: 1px solid black;
+	display:flex;
+	font-family:Arial;
+	margin-left: 180px;
+	padding: 5px 5px 5px;
+	width: 100px;
+}
+
+.line {
+	border-bottom: 1px solid;
+	width: 875px;
+}
+
+.one h6 {
+	
+	font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+	margin-bottom: 15px;
+	margin-top: 20px; 
+}
+
+.one p {
+	display: flex;
+	width: 50%;
+     height: 50px;
+
+}
+
+.one h3  {
+	 
+
+	font-family: 'Bangers', cursive;
+	letter-spacing: 1px;
+	margin-left: 50%;
+	margin-top: -95px;
+	margin-bottom: 15px;
+}
+
+.two p {
+	display: flex;
+	margin-left: 50%;
+	
+}
+
+.three h4 {
+ 
+	font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+	margin-bottom: 15px;
+	margin-top: 20px; 
+	margin-left: -10px;
+}
+
+.three p {
+	display: flex;
+	width: 30%;
```

Comment:
It seems this little bugger was causing some horizontal scrollbars on the webpage. You definitely don't want to leave those in there..!